etsd/users/utils.py

```diff
@@ -4,4 +4,7 @@ def get_authority_users(authority):
     return models.User.objects.filter(authorities=authority)
 
 def get_authority_users_emails(authority):
-    return [usr.email for usr in get_authority_users(authority)]
\ No newline at end of file
+    return [usr.email for usr in get_authority_users(authority)]
+
+def get_admin_emails():
+    return [usr.email for usr in models.User.objects.filter(is_staff=1)]
```

Comment:
The admins are the users that have the `core.admin` permission.

Reply:
I am trying with 
models.User.objects.filter(user_permissions__codename='admin')
and I am getting an empty queryset. I know the codename is right.
Is the superuser not automatically assigned admin permission?

Reply:
Yes, for the superuser it returns True for all permissions (wiithout having them as an entry to the database). Try adding a new user and giving them the core.admin permission to see